<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TcpTestClass</class>
 <widget class="QMainWindow" name="TcpTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TcpTest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>服务器</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="5" column="1" colspan="2">
        <widget class="QSpinBox" name="spinBox_port">
         <property name="minimum">
          <number>5001</number>
         </property>
         <property name="maximum">
          <number>65535</number>
         </property>
         <property name="value">
          <number>6000</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="3">
        <widget class="QTextEdit" name="textEdit_to_client"/>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>IP:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QPushButton" name="pushButton_start_server">
         <property name="text">
          <string>启动服务</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="3">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" rowspan="7">
        <widget class="QTextBrowser" name="textBrowser_server"/>
       </item>
       <item row="4" column="2" colspan="2">
        <widget class="QLabel" name="label_ip">
         <property name="text">
          <string>127.0.0.1</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton_send_to_client">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="pushButton_disconnected_client">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>断开连接</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="2">
        <widget class="QComboBox" name="comboBox_client"/>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>客户端</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="pushButton_send_to_all">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>群发</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="tcp_test.qrc"/>
 </resources>
 <connections/>
</ui>
